lib/zstd.h

```diff
@@ -473,7 +473,7 @@ typedef struct {
 typedef struct ZSTD_CCtx_params_s ZSTD_CCtx_params;
 
 typedef enum {
-    ZSTD_dct_auto=0,      /* dictionary is "full" when starting with ZSTD_MAGIC_DICTIONARY, otherwise it is "rawContent" */
+    ZSTD_dct_auto = 0,      /* dictionary is "full" when starting with ZSTD_MAGIC_DICTIONARY, otherwise it is "rawContent" */
```

Comment:
Could you ensure that the code comments remain vertically aligned ? for the sake of consistency ?

Reply:
sure thing!